administrator/components/com_cookiemanager/src/Table/CookieTable.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cookiemanager
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Cookiemanager\Administrator\Table;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Application\ApplicationHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\String\PunycodeHelper;
+use Joomla\CMS\Table\Table;
+use Joomla\Database\DatabaseDriver;
+use Joomla\Registry\Registry;
+
+/**
+ * Group Table class.
+ *
+ * @since   __DEPLOY_VERSION__
+ */
+class CookieTable extends Table
+{
+
```

Comment:
You should add the `$_supportNullValue` property like you can see it e.g. here: https://github.com/joomla-projects/gsoc21_cookie-manager/blob/4.1-dev/administrator/components/com_contact/src/Table/ContactTable.php#L34-L40 . Of course with the right `@since` tag.